libchihua-msg/include/common/common.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
```

Comment:
This is the same as in `binchihua`. To avoid duplication, I'd say its fine with having it here and include the `libchihua`.